### PR TITLE
Provider: Build and Process

### PR DESCRIPTION
**Pull request by ddollar**
Moves Build and Process into the Provider interface.

- [x]  `convox build` is now scheduled via ECS
- [x]  `convox run` is now scheduled via ECS
- [x]  `convox builds copy` has been removed (use export/import)
- [x]  `convox builds info <id>` displays build info instead of logs
- [x] `convox builds logs <id>` displays build logs
- [x] remove provider logging from build output
- [x] add git fetcher to build
- [x] verify that cron is still working

### CODE REVIEW

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ifkmQ0Qi&#x2F;701-builds-on-ecs"> builds on ecs</a>

File: api/cmd/build/main.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	m, err := manifest.Load(data)
+	if err != nil {
+		return err
+	}
+
+	s := make(chan string)
+
+	go func() {
+		for l := range s {
+			log(l)
```
Comment (MiguelMoll) [inline api/cmd/build/main.go]: Might not be a big deal overall, but we probably want to defer close the channel `s` so this loop will stop nicely.

**Comment by ddollar** [comment]
@MiguelMoll have addresses your comments, please re-review when able

**Comment by MiguelMoll** [comment]
There's a log statement at the end of a build log from the ObjectStore.

```
6d723699a304: Layer already exists
web.BEQJAWQOMDB: digest: sha256:4d513553226772b490c3f9e06f3637272ed176f0a7a7cfcf85054e5fe8809dc0 size: 10130
ns=provider.aws at=ObjectStore key=build/BEQJAWQOMDB/logs public=false state=success elapsed=272.944
Release: RROUEGLJGEF
```

 Might be intentional but I see provider logs being suppressed.

**Comment by ddollar** [comment]
Those logs are now suppressed for me. Are you perhaps on an older build?

**Pull request closed without merging**